src/Adapter/MSTest.CoreAdapter/Execution/TestMethodInfo.cs

```diff
@@ -231,7 +231,7 @@ private TestResult ExecuteInternal(object[] arguments)
 
                     if (result.Outcome != TestTools.UnitTesting.UnitTestOutcome.Passed)
                     {
-                        if (ex.InnerException != null && ex.InnerException is AssertInconclusiveException)
+                        if (ex is UTF.AssertInconclusiveException || ex.InnerException is UTF.AssertInconclusiveException)
```

Comment:
can we please add a null check for `ex.InnerException`?

Reply:
That should not be needed, as is will return false when compared to null. If for clarity, I can add it back in.

Reply:
That's fair. Don't think there would be much of a perf hit.